src/bugsnag/BugsnagErrorBoundary.tsx

```diff
@@ -0,0 +1,38 @@
+// NOTE: do not include any other files here, try to keep this component separate from the rest
```

Comment:
Given this is imported in the main `index.js` it'll be bundled into all of the other dependencies that are included there as well anyway, so this may have little 'real world' impact/benefit.

Reply:
The note is for future maintainers being tempted to add more stuff in it.